source/docs/docs/identity-and-access-management/identity.md

```diff
@@ -0,0 +1,55 @@
+# Identity
+
+## Scope
+
+An identity is a collection of verifiable claims one makes about oneself. In the context of the Infratographer ecosystem, identity primarily focuses on validating the claims a user makes about their identity, with an intentional shift away from management of those claims.
```

Comment:
Should we also specify that an identity pertains to a principal or subject, and is not exclusive to a user? The intention is to assert that we don't care either the type of subject an identity belongs to. That is, user, service user, machine, etc.

Reply:
I think that's a good idea to call out here; it was implied in some other sections but we should be explicit about it.